service/main.go

```diff
@@ -46,6 +47,9 @@ func main() {
 	}
 	level.Info(logger).Log("message", fmt.Sprintf("loading config '%s' completed", env.ConfigFilePath))
 
+	// temp, will rm after config restructure
+	validations.SetImageURIs(env.ImageURIs)
```

Comment:
Had to bring this back here (out of loadConfig) after the envvar move. Consider it an implementation detail / landing spot, that will move with the config improvements.